pyserini/search/pysearch.py

```diff
@@ -368,4 +372,35 @@ def multisearch(self, q: str, k=10) -> List[List[JSimpleNearestNeighborSearcherR
         List(List[JSimpleNearestNeighborSearcherResult])
             List of List of (nearest neighbor) search results (one for each matching id).
         """
-        return self.object.multisearch(JString(q), k)
\ No newline at end of file
+        return self.object.multisearch(JString(q), k)
+
+
+def main(index, topics, output):
+    print(index, topics, output)
+    searcher = SimpleSearcher(index)
+    with open(topics, 'r') as content_file:
+        content = content_file.read()
+    result = re.findall(r'(?<=<top>)(.+?)(?=<desc>)', content, flags=re.S)
```

Comment:
You can load topics directly: https://github.com/castorini/pyserini/blob/master/tests/test_loadtopics.py

So allow something like `--topics robust04`?

Reply:
should I keep allowing the -topics filepath one?

Reply:
interpret it as an id, per above, otherwise interpret it as a path.

Reply:
got it. thanks